pubsub/topics/publish_flow_control.go

```diff
@@ -36,8 +36,9 @@ func publishWithFlowControlSettings(w io.Writer, projectID, topicID string) erro
 	}
 	defer client.Close()
 
-	t := client.Topic(topicID)
-	t.PublishSettings.FlowControlSettings = pubsub.FlowControlSettings{
+	// Make sure to reuse this publisher across publishes.
+	p := client.Publisher(topicID)
```

Comment:
nit: use more expressive variable names instead of single letters. Here, `publisher` instead of `p`.

Reply:
Here and below.